source/common/stats/stats_impl.h

```diff
@@ -103,6 +111,8 @@ struct RawStatData {
 
 private:
   static const size_t DEFAULT_MAX_NAME_SIZE = 127;
+  // total length must be equal to DEFAULT_MAX_NAME_SIZE
+  static const size_t DEFAULT_MAX_STAT_SUFFIX_LENGTH = 67;
```

Comment:
rename to MAX_STAT_SUFFIX_LENGTH?  Since it isn't configurable, I don't think of it as a default; it's the only value.

Reply:
done..